### DISCUSSION TITLE
关于本地歌曲与歌词。

### DISCUSSION THREAD

**Discussion by Perxenic-Acid** [YXL76/cloudmusic-vscode#990] [Q&A]
我尝试将同名的wav文件与lrc文件放在同一个目录并导入，但播放时插件似乎并没有识别到lrc歌词文件，请问如果需要指定歌词文件，我需要如何操作呢？

**Comment by YXL76** [comment]
目前不支持

**Reply by Perxenic-Acid** [reply]
感谢！请问之后有没有做的打算呢？这个插件是我见过最好看的vscode音乐播放方案。
